### PR TITLE
Add options to navigate with / without preview screen

### PR DESCRIPTION
**Pull request by orenco-resonai**

### CODE REVIEW

**Comment by orenco-resonai** [comment]
Will work for TCM only after next SDK is released, and will include PR [15348](https://github.com/resonai/resonai2/pull/15348)

File: sample-apps/communication/main.js [hunk 1]
```diff
@@ -64,16 +64,22 @@ async function init () {
...
+  document.getElementById('navigate-message-nopreview-button').addEventListener('click', () => {
+    onNavigateUsingMessageClick(true)
+  })
   document.getElementById('native-message-button').addEventListener('click', onNativeMessageClick)
   document.getElementById('nearest-poi-button').addEventListener('click', onNearestPoiClick)
 }
 init()
 
-function onNavigateUsingMessageClick () {
+function onNavigateUsingMessageClick (skipPreview) {
```
Comment (yoav-resonai) [inline sample-apps/communication/main.js:78]: `function onNavigateUsingMessageClick ({ skipPreview = false } = {})`

File: sample-apps/communication/main.js [hunk 2]
```diff
@@ -64,16 +64,22 @@ async function init () {
     document.getElementById('navigate-message-button').disabled = true
     document.getElementById('nearest-poi-button').disabled = true
   }
-  document.getElementById('navigate-message-button').addEventListener('click', onNavigateUsingMessageClick)
+  document.getElementById('navigate-message-preview-button').addEventListener('click', () => {
+    onNavigateUsingMessageClick(false)
+  })
+  document.getElementById('navigate-message-nopreview-button').addEventListener('click', () => {
+    onNavigateUsingMessageClick(true)
```
Comment (yoav-resonai) [inline sample-apps/communication/main.js:71]: `onNavigateUsingMessageClick({skipPreview: true})`

File: sample-apps/communication/main.js [hunk 3]
```diff
@@ -64,16 +64,22 @@ async function init () {
     document.getElementById('navigate-message-button').disabled = true
     document.getElementById('nearest-poi-button').disabled = true
   }
-  document.getElementById('navigate-message-button').addEventListener('click', onNavigateUsingMessageClick)
+  document.getElementById('navigate-message-preview-button').addEventListener('click', () => {
+    onNavigateUsingMessageClick(false)
```
Comment (yoav-resonai) [inline sample-apps/communication/main.js:68]: onNavigateUsingMessageClick()

File: sample-apps/communication/main.js [hunk 4]
```diff
@@ -64,16 +64,22 @@ async function init () {
...
+  document.getElementById('navigate-message-nopreview-button').addEventListener('click', () => {
+    onNavigateUsingMessageClick(true)
+  })
   document.getElementById('native-message-button').addEventListener('click', onNativeMessageClick)
   document.getElementById('nearest-poi-button').addEventListener('click', onNearestPoiClick)
 }
 init()
 
-function onNavigateUsingMessageClick () {
+function onNavigateUsingMessageClick ({ skipPreview = false }) {
```
Comment (yoav-resonai) [inline sample-apps/communication/main.js:78]: `{ skipPreview = false } = {}`

**Review by yoav-resonai — APPROVED** [review]

**Pull request merged**